compiler/test/runnable/cov2.d

```diff
@@ -57,5 +68,6 @@ int main(string[] args)
     test1();
     test2();
     test3();
+    test24264();
```

Comment:
Does this check that a file is actually generated

Reply:
I've checked that it does. But the test runner checks cov2.lst, but not ccov2.lst, because I do not know how to make the test runner do that.

Reply:
In general, the test runner does not document things. I usually look for another test case that does something similar, and use it as a template, but could not find one for this case.